setup.py

```diff
@@ -142,16 +143,21 @@ def build_extension(self, ext):
             suffix = "-debug" if self.debug else "-release"
             suffix = conda_suffix + suffix
             preset = ("windows-cl" if platform.system() == "Windows" else platform.system().lower()) + suffix
-        _log_and_run(
+
+        env_var_vcpkg = "ARCTICDB_VCPKG_INSTALL_DIR"
```

Comment:
This doesn't need to be a variable. The name gives no extra information.

Also, I would keep it close to where it's being used.

Reply:
Will do in next commit.